acceptance/Gemfile

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+
+gem 'chef-acceptance', github: 'chef/chef-acceptance'
+gem 'berkshelf'
```

Comment:
Per other comment would like to see all this duplicated code shared with the travis tests, so we should not make any assumptions about ChefDK being installed.